Makefile

```diff
@@ -357,6 +357,8 @@ genpdata:
 check-contrib:
 	@echo Setting contrib at $(CONTRIB_PATH) to use this core checkout
 	make -C $(CONTRIB_PATH) for-all CMD="go mod edit -replace go.opentelemetry.io/collector=$(CURDIR)"
+	# TODO: remove when contrib has been updated with supported version of core
+	make -C $(CONTRIB_PATH) for-all CMD="go get github.com/knadh/koanf github.com/go-yaml/yaml"
```

Comment:
Maybe better, just run `make gotidy` so it fixes any import issue in general, since this will break if we add any new dependency in core.